transmogrifier/sources/ead.py

```diff
@@ -0,0 +1,50 @@
+import logging
+
+from bs4 import Tag
+
+from transmogrifier.sources.transformer import Transformer
+
+logger = logging.getLogger(__name__)
+
+
+class Ead(Transformer):
+    """EAD transformer."""
+
+    @classmethod
+    def get_main_titles(cls, xml: Tag) -> list[Tag]:
+        """
+        Retrieve main title(s) from an EAD XML record.
+
+        Overrides metaclass get_main_titles() method.
+
+        Args:
+            xml: A BeautifulSoup Tag representing a single EAD XML record.
+        """
+        main_titles = []
+        for titlestmt in xml.find_all("titlestmt"):
+            titles = " ".join(
+                string
+                for titleproper in titlestmt.find_all("titleproper")
```

Comment:
The individual `stripped_strings` in a given `titleproper` will still need to be concatenated. What do you think about using a comma `", "` instead of just a space? I haven't looked at enough of the data to know if that makes sense but it seems to for the test fixtures at least...

Reply:
👍 on a comma, and I misread the standard as saying `titlestmt` was repeatable, but given that it's not repeatable, I agree on not concatenating multiple `titleproper`s